src/App.jsx

```diff
@@ -64,6 +64,8 @@ export function App({ gmApiKey, gaTag }) {
   const [userLongitude, setUserLongitude] = useState(0);
   const [currentLocationLoaded, setCurrentLocationLoaded] = useState(false);
 
+  const slug = useRefillSpotSlug();
```

Comment:
Could you check if it's possible to just do
```suggestion
  const slug = getSlug(REFILL_SPOT_ROUTE);
```
to make the code simpler?